frontend/src/views/FuelCodes/FuelCodes.jsx

```diff
@@ -51,9 +40,8 @@ const FuelCodesBase = () => {
   }
 
   const handleRowClicked = (params) => {
-    if (!isAuthorized) return
     navigate(
-      ROUTES.FUELCODES_VIEW.replace(':fuelCodeID', params.data.fuelCodeId)
+      ROUTES.FUELCODES_EDIT.replace(':fuelCodeID', params.data.fuelCodeId)
```

Comment:
How to add a new Fuel Code? I see the route `/add-fuel-code` is not longer available

Reply:
{
        path: ROUTES.FUELCODES_ADD,
        element: <AddEditFuelCode />,
        handle: { title: 'Add Fuel Code' }
      },
      {
        path: ROUTES.FUELCODES_EDIT,
        element: <AddEditFuelCode />,
        handle: { title: 'Fuel Code' }
      },
      
      Add Fuel Code is the same component but no ID on the URL.